internal/motor/config.go

```diff
@@ -7,7 +7,7 @@ import (
 
 func setupDefault() (*MotorNode, error) {
 	// Create Client instance
-	c := client.NewClient(client.ConnEndpointType_BETA)
+	c := client.NewClient(client.ConnEndpointType_LOCAL)
```

Comment:
Can we read this from env?

Reply:
or config or something

Reply:
actually did not mean to make this change, but we should have this be configurable before `dev`.